Sources/Remote Logging/Crash Logging/SentrySDK+CurrentHub.swift

```diff
@@ -31,6 +29,12 @@ extension SentrySDK {
     /// - Note: Once we'll migrate to version 7.x, this will return `.none` because `currentHub`
     /// will no longer be available.
     private static func _currentHub() -> SentryHub? {
-        currentHub()
+        let currentHubSelector = #selector(getter: SentrySDKInternalMethods.currentHub)
+
+        guard SentrySDK.responds(to: currentHubSelector) else {
+            return nil
+        }
+
+        return SentrySDK.perform(currentHubSelector).takeUnretainedValue() as? SentryHub
```

Comment:
Should we update the comment on this method, now that it won't return `.none` anymore?

Reply:
Updated